core-web/libs/data-access/src/lib/dot-containers/dot-containers.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Observable } from 'rxjs';
+
+import { Injectable } from '@angular/core';
+
+import { pluck } from 'rxjs/operators';
+
+import { CoreWebService, DotRequestOptionsArgs } from '@dotcms/dotcms-js';
+import { DotContainerEntity } from '@dotcms/dotcms-models';
+
+export const CONTAINER_API_URL = '/api/v1/containers/';
+
+/**
+ * Provide util methods to handle containers in the system.
+ * @export
+ * @class DotContainersService
+ */
+@Injectable()
+export class DotContainersService {
```

Comment:
There was one very similar to this, with more methods in dotcms-ui. Felt like it was a good idea to create a new one in the data-access lib. Would appreciate confirmation.